src/components/Button/styles.module.css

```diff
@@ -0,0 +1,34 @@
+.button {
+    height: 52px;
+    border-radius: 6px;
```

Comment:
Uma boa prática no geral é a gente usar `rem` ao invés de `px` como unidade.
Aqui tem um bom artigo sobre essa discussão se tiver interesse:
https://www.seeratawan.me/blog/why-you-should-move-from-px-to-rem-in-css/#:~:text=Rems%20Provide%20Major%20Accessibility%20Advantages&text=If%20a%20user%20has%20trouble,or%20override%20the%20text%20size.

Reply:
ajustados todos para rem